Header header

geometry_msgs/PoseStamped pose
string name
string group